src/app/core/utils/date-utils.ts

```diff
@@ -4,8 +4,67 @@
  * Can be used to ensure dates are displayed identically across different timezones when stringified in JSON payloads.
  * @param date date to convert
  */
+import {addDays, subDays} from 'date-fns';
 import {format} from 'date-fns-tz';
 
 export function toGmtZeroHourDate(date: Date): Date {
   return new Date(format(date, 'yyyy-MM-dd'));
 }
```

Comment:
Is this doc relating to `toGmtZeroHourDate`? Even if not, why is this above the imports?

Reply:
fixed